test-case/check-playback.sh

```diff
@@ -94,6 +102,17 @@ do
                     dmesg > $LOG_ROOT/aplay_error_${dev}_$i.txt
                     dloge "aplay on PCM $dev failed at $i/$loop_cnt."
                     exit 1
+                else
+                    #sleep ${sleep_lst[@]} seconds before running aplay, then aplay can be run under dsp active or suspended status
+                    if [[ $(sof-dump-status.py --dsp_status 0) != "unsupported" ]] && [ "$sleep_ctl" -eq 1 ]; then
+                        sleep_time=${sleep_lst[$((j++ % sleep_len))]}
```

Comment:
@ClarexZhou what are we doing in this part (j++ % sleep_len)? Can we not make it simply use a random number between 1 and sleep_len to make this intuitive?

Reply:
let's be careful with random numbers: https://github.com/thesofproject/sof-test/issues/301

Random with fixed seed: good for CI
Random with variable but assignable seed: good for stress tests, still repeatable
Random without controlled seed: super stress tests, not repeatable.

Reply:
@plbossart we're not randomizing the sleep times themselves but just the index of which sleep time to pick from the pre-defined list. So it should be fine no?

Reply:
The net result is just as non-repeatable.